### PR TITLE
Fix unit tests on Windows

### PR DESCRIPTION
**Pull request by samolisov**
Fix the unit tests of the project in order to successfully run on Windows (there were few problems with Windows' file path separator and a disk name letter). I'm not sure whether the committers are interested in the ability to build the project on Windows but if you have any question regarding the PR, please, be free to ask them.

### CODE REVIEW

File: generator/api/src/test/java/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest.java [hunk 1]
```diff
@@ -1,23 +1,22 @@
 package io.fabric8.maven.generator.api.support;
```
Comment (rohanKanojia) [inline generator/api/src/test/java/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest.java]: You forgot to add license header in this file ;)

**Comment by rohanKanojia** [comment]
Hi Pavel,

Sorry for the long delay.Thanks a lot for this :+1 . I saw that stackoverflow post and found it relevant, we should have to in for sure.
I see this method:
```
private static String decodeUrl(String url) { return URLDecoder.decode(url, "UTF-8"); }
```
copy pasted in other places also in code, namely `ClassUtilTest` and `FatJarDetectorTest`. Could you please replace those methods also with your `getAbsolutePath()` call?

File: generator/api/src/test/java/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest.java (hunk 1 above)
Comment (samolisov) [reply generator/api/src/test/java/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest.java]: I'm not an initial author of this file, but I'll add the license header.

Comment (rohanKanojia) [reply generator/api/src/test/java/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest.java]: oops

**Comment by samolisov** [comment]
Hello @rohanKanojia 

I've replaced all `decodeUrl()` calls to `FileUtil.getAbsolutePath()` ones.
Also some eclipse-related files have been added to `.gitignore`.

**Comment by rohanKanojia** [comment]
@samolisov : Thanks. Looks good. Could you please squash your commits?

**Comment by samolisov** [comment]
@rohanKanojia Done!

**Pull request merged**